tests/integration_tests/build/test_coverage.py

```diff
@@ -23,7 +23,7 @@
 # this contains the frequency while on AMD it does not.
 # Checkout the cpuid crate. In the future other
 # differences may appear.
-COVERAGE_DICT = {"Intel": 85.35, "AMD": 84.63, "ARM": 82.87}
+COVERAGE_DICT = {"Intel": 85.25, "AMD": 84.63, "ARM": 82.87}
```

Comment:
I see the coverage drop. Are there any rust tests that cover the VCPU emulation loop? Could we add these new event types to a test?

Reply:
There are no unit tests as far as I know that exercise the emulation loop in Firecracker (would be out of the scope fo this PR to add the framework for that here). What we can do as far as testing this functionality is make use of this [integration test](https://github.com/firecracker-microvm/firecracker/blob/master/tests/integration_tests/functional/test_shut_down.py#L58) to test the expected message.